### ISSUE TITLE
Egui Context panics

### ISSUE THREAD

**Issue by DasLixou** [Plonq/bevy_panorbit_camera#79]
I get this error after upgrading to bevy 0.14
```rs
`EguiContexts::ctx_for_window_mut` was called for an uninitialized context (window Entity { index: 92, generation: 5 }), make sure your system is run after [`EguiSet::InitContexts`] (or [`EguiStartupSet::InitContexts`] for startup systems)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in system `bevy_panorbit_camera::egui::check_egui_wants_focus`!
```
I don't have any clue why it doesn't work now but it worked before the upgrade and I haven't changed anything else.
I dynamically create a new window and now, `bevy_panorbit_camera` just intervenes and crashes because of a missing egui context.

**Comment by Plonq** [comment]
Hi, can you confirm you've updated bevy_panorbit_camera to 0.19? And if so, can you test running the egui example? `cargo  run --example egui`

**Comment by DasLixou** [comment]
`bevy_panorbit_camera = { version = "0.19.0", features = ["bevy_egui"] }`

**Comment by DasLixou** [comment]
![image](https://github.com/Plonq/bevy_panorbit_camera/assets/82600264/8e4f9ce2-0ee1-455e-be38-7fc273445a51)
Example works

**Comment by DasLixou** [comment]
Maybe I have to sort my window creation system to be run after your code? Or any other idea?

**Comment by Plonq** [comment]
The error suggests that `ctx_for_window_mut` is called before the `EguiSet::InitContexts` system set. The only instance of `ctx_for_window_mut` in this crate is in the `egui_multiple_windows` example. Additionally, I run the egui system `check_egui_wants_focus` after `EguiSet::InitContexts` explicitly. Which means it's probably a problem with your code. If you provide a minimal reproducible example I can have a look at it.

**Comment by DasLixou** [comment]
I'll try to make something reproducable

**Comment by DasLixou** [comment]
https://gist.github.com/DasLixou/81632aceba0990fb2229a094387a7b72

**Comment by DasLixou** [comment]
When pressing and releasing space, the app crashes

**Comment by Plonq** [comment]
Thanks for the repro. I've fixed it in 0.19.1, by using `try_ctx_for_window_mut` instead of `ctx_for_window_mut` to gracefully handle cases where the egui context doesn't exist for a given window.
